### PR TITLE
Add multi-threading example + complex AOI to timeseries generation example

### PR DESCRIPTION
**Pull request by leothomas**
Screenshots: 

- Complex AOI: France (incl. overseas territories) 
image

- Plot with min/max/mean/stddev: 
image

- Multi-threaded example/explanations:  
image

### CODE REVIEW

File: timeseries-stac-api.ipynb [hunk 1]
```diff
@@ -20,17 +20,16 @@
...
+    "import folium \n",
     "import json\n",
     "import requests as re\n",
     "import matplotlib.pyplot as plt\n",
     "import pprint\n",
     "import time\n",
     "\n",
-    "STAC_API_URL = \"https://staging-stac.delta-backend.com\"\n",
-    "RASTER_API_URL = \"https://staging-raster.delta-backend.com\""
+    "STAC_API_URL = \"https://staging-stac.delta-backend.xyz\"\n",
```
Comment (abarciauskas-bgse) [inline timeseries-stac-api.ipynb]: fyi @leothomas I think these urls should be `.com`

**Comment by abarciauskas-bgse** [comment]
@leothomas and I discussed updating the URL's to .com, adding titles, adding to the TOC the new notebook(s) and also keeping the existing simple bounding box example

@leothomas let me know when you want me to re-review and thanks agin for all your work on these awesome notebooks!

**Comment by leothomas** [comment]
@abarciauskas-bgse this is ready for a re-review. 
- [x] Updated the timeseries notebook to include a simple bounding box example + complex AOI example
- [x] Switched URL's to `staging-*.delta-backend.com`
- [x] Updated the quarto TOC to include the timeseries notebook

**Review by abarciauskas-bgse — APPROVED** [review]
LGTM, I tried to run things locally but am getting an error (unrelated to this) from the Raster API

**Pull request merged**